lambdas/data-migration2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@cumulus/data-migration2",
+  "version": "3.0.0",
+  "description": "A Lambda function used for doing data migrations of executions",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build-lambda-zip": "cd dist/webpack && node ../../../../bin/zip.js lambda.zip index.js",
+    "package": "npm run clean && npm run prepare && npm run webpack && npm run build-lambda-zip",
+    "test": "../../node_modules/.bin/ava",
+    "test:coverage": "../../node_modules/.bin/nyc npm test",
+    "prepare": "npm run tsc",
+    "tsc": "../../node_modules/.bin/tsc",
+    "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
+    "webpack": "../../node_modules/.bin/webpack"
+  },
+  "ava": {
+    "files": [
+      "tests/**/*.js"
+    ],
+    "timeout": "15m"
+  },
+  "dependencies": {
+    "@cumulus/api": "3.0.0",
+    "@cumulus/aws-client": "3.0.0",
+    "@cumulus/common": "3.0.0",
+    "@cumulus/db": "3.0.1-alpha.1",
+    "@cumulus/errors": "3.0.0",
+    "@cumulus/logger": "3.0.0",
+    "@cumulus/types": "3.0.0",
+    "knex": "0.21.13",
+    "lodash": "^4.17.20",
+    "pg": "^8.3.0"
+  },
+  "devDependencies": {
+    "@cumulus/test-data": "3.0.0",
+    "@cumulus/data-migration1": "3.0.0"
```

Comment:
I'm using `data-migration1` in tests. I need to be able to migrate Collections and Async Operations to test fk relationships with executions. Happy to move these somewhere more common if there's a better place. The requires are in test-executions.js:

```
const { migrateAsyncOperationRecord } = require('@cumulus/data-migration1/dist/lambda/async-operations');
const { migrateCollectionRecord } = require('@cumulus/data-migration1/dist/lambda/collections');
```

Reply:
Lies! This dependency will be removed after a conversation below about not creating these dynamo records in test.